### PR TITLE
Minor: make it easier to find fix instructions when `cargo fmt` on parquet fails

### PR DESCRIPTION
**Pull request by alamb**
# Which issue does this PR close?

Closes #.

# Rationale for this change
 
While working on https://github.com/apache/arrow-rs/pull/6840 the `fmt` ci check failed on parquet. For example: https://github.com/apache/arrow-rs/actions/runs/12361198145/job/34497838320

```
Run cargo fmt -p parquet -- --check --config skip_children=true `find . -name "*.rs" \! -name format.rs`
Diff in /__w/arrow-rs/arrow-rs/parquet/src/arrow/schema/mod.rs:1[7](https://github.com/apache/arrow-rs/actions/runs/12361198145/job/34497838320#step:7:8)75:
             Field::new("decimal256", DataType::Decimal256(39, 2), false),
         ];
         let arrow_schema = Schema::new(arrow_fields);
-        let converted_arrow_schema = ArrowSchemaConverter::new()
-            .convert(&arrow_schema)
-            .unwrap();
+        let converted_arrow_schema = ArrowSchemaConverter::new().convert(&arrow_schema).unwrap();
 
         assert_eq!(
             parquet_schema.columns().len(),
Error: Process completed with exit code 1.
```

However, the exact recipe to fix this was not clear (you have to run a certain command within the arquet directory)

# What changes are included in this PR?

1. Add the commands needed to fix CI as comments in the script so they are visible in the failure

Example https://github.com/apache/arrow-rs/actions/runs/12361621609/job/34499177568?pr=6886

Screenshot 2024-12-16 at 4 30 04 PM

# Are there any user-facing changes?

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -101,12 +101,13 @@ jobs:
...
-        # Many modules in parquet are skipped, so check parquet separately. If this check fails, run:
-        #   cargo fmt -p parquet -- --config skip_children=true `find ./parquet -name "*.rs" \! -name format.rs`
-        # from the top level arrow-rs directory and check in the result.
+        # Many modules in parquet are skipped, so check parquet separately
         # https://github.com/apache/arrow-rs/issues/6179
         working-directory: parquet
-        run: cargo fmt -p parquet -- --check --config skip_children=true `find . -name "*.rs" \! -name format.rs`
+        run: |
+          # if this fails, run this from the parquet directory:
+          # cargo fmt -p parquet -- --config skip_children=true `find ./parquet -name "*.rs" \! -name format.rs`
```
Comment (etseidl) [inline .github/workflows/rust.yml:109]: If running from the parquet directory, then just '.' for the find path:
```suggestion
          # if this fails, run this from the parquet directory:
          # cargo fmt -p parquet -- --config skip_children=true `find . -name "*.rs" \! -name format.rs`
```

**Review by etseidl — APPROVED** [review]
🚀 Nice. Just a small correction.

**Comment by alamb** [comment]
> 🚀 Nice. Just a small correction.

Nice catch -- thank you

**Review by tustvold — APPROVED** [review]
Is there a reason we have such an arcane incantation?

**Comment by alamb** [comment]
> Is there a reason we have such an arcane incantation?

Because for some reason `cargo fmt` doesn't format the files in the parquet crate. 

Added in https://github.com/apache/arrow-rs/commit/25e196977fd4f2fdd9ad25b3d3936d355ce27d46 / https://github.com/apache/arrow-rs/pull/6328

**Comment by etseidl** [comment]
> > Is there a reason we have such an arcane incantation?
> 
> Because for some reason `cargo fmt` doesn't format the files in the parquet crate.
> 
> Added in [25e1969](https://github.com/apache/arrow-rs/commit/25e196977fd4f2fdd9ad25b3d3936d355ce27d46) / #6328

In particular, things like https://github.com/apache/arrow-rs/blob/54dccadccc6b599b93a46aef5f03dfd4d9b7349e/parquet/src/lib.rs#L137-L138 cause entire modules to be skipped by `cargo fmt`. I agree the incantation is not desirable, and was meant to be a stop-gap until a better solution can be found.

**Comment by alamb** [comment]
> cause entire modules to be skipped by `cargo fmt`. I agree the incantation is not desirable, and was meant to be a stop-gap until a better solution can be found.

Filed https://github.com/apache/arrow-rs/issues/6897 to track

**Pull request merged**